Sources/FacebookCoreTests/GraphApiDebugParameterTests.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+//
+// You are hereby granted a non-exclusive, worldwide, royalty-free license to use,
+// copy, modify, and distribute this software in source code or binary form for use
+// in connection with the web services and APIs provided by Facebook.
+//
+// As with any software that integrates with the Facebook platform, your use of
+// this software is subject to the Facebook Developer Principles and Policies
+// [http://developers.facebook.com/policy/]. This copyright notice shall be
+// included in all copies or substantial portions of the software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// swiftlint:disable multiline_arguments
+
+@testable import FacebookCore
+import XCTest
+
+class GraphApiDebugParameterTests: XCTestCase {
+
```

Comment:
These tests feel to me like they’re testing Apple’s code much more than our own (i.e. check if `allCases` and `rawValues` for enums work as they’re supposed to). Thoughts?

Reply:
Disagree. Both are beneficial to ensure that we're covering all cases should the enum ever change. Super easy to mass-rename all instances in code using Xcode, but the testing against rawValues ensures that we change with full disclosure.

Reply:
I also am a little bit trying to encourage the idea that if something needs changing it's a conscious decision and not just an easy hack. Sort of the guardrails approach.